src/server/game/Grids/NGrid.cpp

```diff
@@ -23,7 +23,7 @@ i_unloadActiveLockCount(0), i_unloadExplicitLock(false), i_unloadReferenceLock(f
 {
 }
 
-GridInfo::GridInfo(time_t expiry, bool unload /*= true */) : i_timer(expiry), vis_Update(0, irand(0, DEFAULT_VISIBILITY_NOTIFY_PERIOD)),
+GridInfo::GridInfo(std::chrono::seconds expiry, bool unload /*= true */) : i_timer(expiry.count()), vis_Update(0, irand(0, DEFAULT_VISIBILITY_NOTIFY_PERIOD)),
```

Comment:
you dont need to specify this as seconds, std::chrono is fine

Reply:
I can't just write std::chrono, that gives me an error. I believe the alternative to std::chrono::seconds would be std::chrono::duration. Correct me if I am wrong 👀

Reply:
then it is fine, my memory can also be failing me. I apologize then